ext/iperf3/Makefile.am

```diff
@@ -0,0 +1,102 @@
+lib_LTLIBRARIES         = libiperf.la                                   # Build and install an iperf library
```

Comment:
I wonder if we should introduce all these autotools related files. Do you use them for anything (testing or something)?

Reply:
We'll keep autotools input files.